Sources/SwiftDocC/Model/Rendering/Content/RenderContentMetadata.swift

```diff
@@ -18,6 +18,8 @@ public struct RenderContentMetadata: Equatable, Codable {
     public var title: String?
     /// An optional custom abstract.
     public var abstract: [RenderInlineContent]?
+    /// An optional identifier for the device frame that should wrap this element.
```

Comment:
Should we add the same experimental-feature mention to this doc comment that we have for the `DirectiveArgumentWrapped ` property?
```
/// This is an experimental feature – any device frame specified here
/// must be defined in the `theme-settings.json` file of the containing DocC catalog.
```